src/main/java/uk/gov/hmcts/darts/retention/model/CaseRetentionConfidenceReason.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.retention.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+import java.util.List;
+
+@Data
+@Builder
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CaseRetentionConfidenceReason {
```

Comment:
Should include validation in the JSON object

Reply:
What do you mean?

Reply:
Things like @NotNull, @Pattern, @Length etc

Reply:
No there are no rules on it although the next ticket will define the date format.